lib/utilities/monitoringHandler.js

```diff
@@ -64,6 +64,31 @@ const httpResponseSizeBytes = new client.Summary({
     help: 'Cloudserver HTTP response sizes in bytes',
 });
 
+const quotaEvaluationDuration = new client.Histogram({
```

Comment:
could be nice to have a condition to disable those metrics for S3C if it doesn't need this metrics

Reply:
I think we will want to have this feature in S3C. For now I am only targeting Zenko branches. We have quotas (account only, and limited) in S3C today though.

Reply:
what is the point of disabling the metrics?
if the metric is not "used" in the code (e.g. never `observed()`), no time series will be generated...

Reply:
Yes, for metrics that have labels like this one, no time series is generated, but for other metrics without label (like `s3_cloudserver_quota_buckets_count`) they are initialized at 0 and will return 0 at every scrape even if unused, so prometheus will keep storing 0 (every 15s on the ring) and increase storage size.

As the prometheus/thanos DB can take a lot of space, it would be nice to not scrape unused metrics for the feature that's not enabled.

Reply:
Is this approach answering your concerns? https://github.com/scality/cloudserver/pull/5556/files#diff-3daa33d35f5375a7070d74a0b174c01ab8b982392db4025ebc49cf3690aa16bcR75

Reply:
Yep exactly. And we could do the same in scuba repo for mongo specific metrics